client-core/src/wallet/syncer_logic.rs

```diff
@@ -274,6 +281,23 @@ mod tests {
             .collect()
     }
 
+    fn outgoing_staking_transaction() -> Transaction {
+        Transaction::DepositStakeTransaction(DepositBondTx {
+            inputs: vec![TxoPointer {
+                id: [3; 32],
```

Comment:
why fill with value 3, some explanation  would be good

Reply:
it is unit test, I think it Ok here using `32`. Because `[0;32]` and `[1;32]` already used in the other type of transactions, and it should be unique here to avoid confliction, so I use `[3; 32]`